GitHub-Test-DHprojects/erinMooney-GitTest4.md

```diff
@@ -0,0 +1,7 @@
+# Response to One of the Other Student Works Websites
+
+By: Erin Mooney
+
+This is a really cool project! I am very interested in languages and how they translate. I also like the set-up of the website, as well as the color pallet! They also have a nice variety of bands especially because they had such a small group.
+
+The About tab covers a lot of ground, such as what their project is about, what K-Pop is, and about themselves. Their Group Profiles are very clean, organized, and readable. Each profile is short, to the point, and very informative. The Song section literally has anything you could need to know, including a section that highlights different singer’s parts based on who you choose. Their analysis page also looks great and reads well. My only problem is with the ‘Popularity & Brand Reputation Ranking (Nov. 2020)’ graph because the setup is misleading (small numbers are on top, big numbers are on the bottom which is the opposite of how graphs are usually constructed). Looking at their ‘Our Code’ section was very interesting because they go into depth as to why they made certain decisions. They also show bits of their code (it looks so clean!) which supports what they are describing in the text around the picture.  
```

Comment:
Although I did not write my discussion responses on this website; I did get to check it out and see the work that the students put into the project. Like you said their website was visually appealing besides the little mishap with the graph. I also caught that when looking through. It reminded me of another website that had graphs with no labels at all. Good eye! The code section on the webpage is really cool because it gives us a chance to see what we may be doing ourselves in the coming weeks of the semester!